tests/data/current_time.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WordpressStubs\Tests;
+
+use stdClass;
+
+use function current_time;
+use function PHPStan\Testing\assertType;
+
+// Integer types
+assertType('int', current_time('timestamp'));
+assertType('int', current_time('U'));
+
+// String types
+assertType('string', current_time('mysql'));
+assertType('string', current_time('Hello'));
+
+// Unknown types
+assertType('int|string', current_time($_GET['foo']));
+assertType('int|string', current_time(get_option('date_format')));
```

Comment:
I think we can drop these as well. This will always result in a fatal error if `$_GET['foo']` and `get_option('date_format')` are not string.

If we want to include an unknown string we should do this
```php
/** @var string $string */
$string = null;
assertType('int|string', current_time($string));
```